CONTRIBUTING.md

```diff
@@ -265,14 +265,15 @@ This website will host all approved and published versions of all protocols.
 For adding a **pre-existing version of a protocol that was written in `docx` format**, follow the steps to [create a new protocol](#workflow-new), except in the second step:
 
 -   a subject-matter specialist uses `protocolhelper::create_protocol()` (or one of its shortcut functions `create_sfp()` or `create_spp()`) to convert the `docx` protocol to Rmarkdown files. See section [From an existing docx protocol](#from-an-existing-docx-protocol).
--   the old protocol-code from the pre-existing `docx` protocol will likely not correspond with the new protocol-code. Mention this in the protocol-specific `NEWS.md` file. Use the new protocol-code to create a new branch with `checklist::new_branch()`
+-   the old protocol-number from the pre-existing `docx` protocol can be re-used and specified in through the `protocol_number` argument. You can check the list of pre-existing protocol numbers using `protocolhelper:::reserved_codes`. Mention in the protocol-specific `NEWS.md` file that your new protocol was created starting from this pre-existing protocol. Use the new protocol-code to create a new branch with `checklist::new_branch()`
```

Comment:
```suggestion
-   the old protocol-number from the pre-existing `docx` protocol can be re-used and specified in through the `protocol_number` argument. You can check the list of pre-existing protocol numbers using `protocolhelper:::reserved_codes()`. Mention in the protocol-specific `NEWS.md` file that your new protocol was created starting from this pre-existing protocol. Use the new protocol-code to create a new branch with `checklist::new_branch()`
```

Reply:
The brackets should not be added, because it's not a function. It is an internal dataset.